weg-li/Features/Mail/Model/District.swift

```diff
@@ -8,9 +8,9 @@
 
 import Foundation
 
-struct Publicaffairsoffice: Decodable {
+struct District: Decodable {
     let name: String
-    let zipCode: String?
```

Comment:
Since the recognition also works with the `zipCode` it makes totally sense!

Reply:
Is there a matching process implemented on the backend? Otherwise it works be cool to also have the coordinates for matching I think

Reply:
Does the parsing still work when this is not optional? Think the list did not provide a zipcode for all entries

Reply:
We should be able to easily convert the geo coordinate into an address with the help of https://developer.apple.com/documentation/corelocation/clgeocoder/1423621-reversegeocodelocation and use the `zipCode` to find out the email address of the district.

Reply:
If the json includes districts without a zip or email we should regenerate it. There is also more meta-info on the district that we can use, ie the coordinates.